src/styles/_shell.scss

```diff
@@ -92,12 +92,15 @@
   }
 }
 
-.player.casting,
-.player.metadata-loaded.hover,
-.player.state-paused,
-.player.state-idle,
-.player.ad-break,
-.player.menu-active,
-.player.overlay-active {
+.player {
+  .pre-playback-play-overlay,
```

Comment:
```suggestion
  &.pre-playback-play-overlay,
```

Reply:
they are not sitting on the same element class attribute (like the rest)